src/scopes/grants/region.js

```diff
@@ -0,0 +1,9 @@
+import { Op } from 'sequelize';
```

Comment:
I'd be on the lookout for slow queries. The Activity Report scopes slowed down queries a ton. I think this is a fine way to start out these queries but you may find you need to update them in a similar fashion as the Activity Report scopes.